content/en/blog/news/first-post/index.md

```diff
@@ -3,15 +3,117 @@ date: 2024-02-28
-author: Fermyon Technologies ([@fermyontech](https://twitter.com/fermyontech))
+  Bringing the next wave of hyper-efficient serverless to Kubernetes.
+author: The SpinKube Team ([@SpinKube](https://mastodon.social/@SpinKube))
 resources:
   - src: "**.{png,jpg}"
     title: "Image #:counter"
 ---
+Today we're introducing SpinKube - an open source platform for efficiently 
+running Spin-based WebAssembly applications on Kubernetes. Built with love by 
+folks from Microsoft, SUSE, LiquidReply, and Fermyon.
 
-# Introducing SpinKube
+SpinKube combines the application lifecycle management of the [Spin Operator][spin-operator], the efficiency of the [containerd-shim-spin][containerd-shim-spin], and the node lifecycle management of the forthcoming [runtime-class-manager][runtime-class-manager] (formerly KWasm) to provide an excellent developer and operator experience alongside excellent density and scaling characteristics regardless of processor architecture or deployment environment.
 
+>
+> TL;DR? Check out the [quickstart][quickstart] to learn how to deploy
+> WebAssembly to Kubernetes today.
+> 
 
+## Why Serverless?
 
+Containers and Kubernetes revolutionized the field of software development and operations. A unified packaging and dependency management system that was flexible enough to allow most applications to become portable - to run the same versions locally as you did in production - was a desperately needed iteration from the Virtual Machines that came before.
 
+But while containers give us a relatively light weight abstraction layer, they come with a variety of issues:
+
+- patching system dependencies like OpenSSL needs to happen in every image independently
+- managing access to shared resources like CPU/memory/disk is hard, and leads to over-provisioning resources which results in low-density and wasteful compute
+- containers are often much bigger than they need to be - including far more system dependencies than are required to run an application.
+
+The desire to fix many of these issues is what led to the "first" generation of Serverless.
+
+>
+> Borrowing ideas from CGI, Serverless apps are not written as persistent servers. Instead Server-less code is code that responds to events (such as HTTP requests), but does not run as a daemon process listening on a socket. The networking part of serverless architectures is relegated to the underlying infrastructure.
+> 
+
+First generation serverless runtimes (such as AWS Lambda, Google Cloud Functions, Azure Functions, and their Kubernetes counterparts OpenWhisk and KNative) are all based on the principle of running a VM or container per function. While this is very flexible, neither compute type is designed to start quickly, an elaborate dance of pre-warming compute capacity and loading a workload just-in-time made the first generation of Serverless slow and inefficient.
```

Comment:
Added a sentence or two to clarify what we mean by "flexibility" (assuming it's app dev model, but feel free to correct)

```suggestion
First generation serverless runtimes (such as AWS Lambda, Google Cloud Functions, Azure Functions, and their Kubernetes counterparts OpenWhisk and KNative) are all based on the principle of running a VM or container per function. While this afforded flexibility for application developers, complexity was introduced to platform engineers as neither compute type is designed to start quickly. Platform engineers operating these serverless platforms became responsible for an elaborate dance of pre-warming compute capacity and loading a workload just-in-time, making for a difficult tradeoff in cold start performance and cost. The result? A slow and inefficient first generation of Serverless.
```